tests/unit/lms/services/canvas_api/client_test.py

```diff
@@ -170,6 +176,31 @@ def test_list_files(self, canvas_api_client, http_session):
             timeout=Any(),
         )
 
+    @pytest.mark.usefixtures("list_files_response")
+    @pytest.mark.parametrize("file_id", [1, "1"])
```

Comment:
The link between the number here and the fixture is a little subtle. But given:

 * It has to be an int
 * You want to test variations

I'm not really sure what you could / should do about it. The only thing I can think of is have `FILE_ID` as a constant and then do `[FILE_ID, str(FILE_ID)]` but ain't nobody got time for that.

Reply:
Yeah I thought (briefly) about but it wasn't easy enough so I get sad and gave up. The constant was also my only idea